packages/saved-scripts/src/saved-scripts-list-item.tsx

```diff
@@ -19,47 +19,48 @@ import {
 } from './saved-scripts.styled';
 
 export interface ISavedScriptsListItemProps {
-  isStatic?: boolean,
-  script: IScript,
-  onSelectScript: AnyFunc,
-  onExecScript: AnyFunc,
-  onUpdateScript: AnyFunc,
-  onRemoveScript: AnyFunc,
-  connectDragSource?: AnyFunc
+  isStatic?: boolean;
+  script: IScript;
+  isRelateScripts?: boolean;
+  onSelectScript: AnyFunc;
+  onExecScript: AnyFunc;
+  onUpdateScript: AnyFunc;
+  onRemoveScript: AnyFunc;
+  connectDragSource?: AnyFunc;
 }
 
 export default DragSource<ISavedScriptsListItemProps>(
   ({ script }) => script.path,
   {
     beginDrag: ({ script }) => script,
   },
-  (connect) => ({
+  connect => ({
     connectDragSource: connect.dragSource(),
   }),
 )(SavedScriptsListItem);
 
 function SavedScriptsListItem({
   isStatic,
   script,
+  isRelateScripts,
   onSelectScript,
   onExecScript,
   onUpdateScript,
   onRemoveScript,
   connectDragSource,
 }: ISavedScriptsListItemProps) {
   const displayName = getScriptDisplayName(script);
-  const [isEditing, nameValue, setIsEditing, setLabelInput] = useNameUpdate(
-    displayName,
-    name => onUpdateScript(script, { name }),
+  const [isEditing, nameValue, setIsEditing, setLabelInput] = useNameUpdate(displayName, name =>
+    onUpdateScript(script, { name }),
   );
   const [blurRef] = useCustomBlur(() => setIsEditing(false));
 
   return (
-    <SavedScriptsListItemMain ref={blurRef} className='saved-scripts-list-item'>
-      {isEditing ? (
+    <SavedScriptsListItemMain ref={blurRef} className="saved-scripts-list-item">
+      {isEditing && !isRelateScripts ? (
```

Comment:
this could do with improving / cleaning up but I didnt want to leave a large footprint at the moment

Reply:
ideally would be good to get rid of edit icon on hover and just have the delete icon appear on hover. But this is liable to change in upcoming work.